src/topology/algebra/multilinear.lean

```diff
@@ -133,18 +132,19 @@ def prod (f : continuous_multilinear_map R M₁ M₂) (g : continuous_multilinea
   (f : continuous_multilinear_map R M₁ M₂) (g : continuous_multilinear_map R M₁ M₃) (m : Πi, M₁ i) :
   (f.prod g) m = (f m, g m) := rfl
 
-/- If `R` and `M₃` are implicit in the next definition, Lean is never able to infer them, even
-given `g` and `f`. Therefore, we make them explicit. -/
-variables (R M₃)
-
-/-- If `g` is continuous multilinear and `f` is continuous linear, then `g (f m₁, ..., f mₙ)` is
-again a continuous multilinear map, that we call `g.comp_continuous_linear_map f`. -/
+/-- If `g` is continuous multilinear and `f` is a collection of continuous linear maps,
+then `g (f₁ m₁, ..., fₙ mₙ)` is again a continuous multilinear map, that we call
+`g.comp_continuous_linear_map f`. -/
 def comp_continuous_linear_map
-  (g : continuous_multilinear_map R (λ (i : ι), M₃) M₄) (f : M₂ →L[R] M₃) :
-  continuous_multilinear_map R (λ (i : ι), M₂) M₄ :=
-{ cont := g.cont.comp $ continuous_pi $ λj, f.cont.comp $ continuous_apply _,
-  .. g.to_multilinear_map.comp_linear_map R M₃ f.to_linear_map }
-variables {R M₃}
+  (g : continuous_multilinear_map R M₁' M₄) (f : Π i : ι, M₁ i →L[R] M₁' i) :
```

Comment:
From what I've seen elsewhere, should wrapped arguments be unindented? @Vierkantor, you called me out on this before I think

Reply:
If I parse the statement correctly, this should be indented because it's a continuation of the `def` line. If it was a `struct`, then each field can appear on its own line.

Reply:
Ah, I was of the understanding that the format was
```lean
def something_long (a : A)
(b : A) : 
  a = b :=
begin
   -- indented proof
end
```
that is, with the actual statement indented, but the arguments unindented. Is there a document somewhere that codifies the mathlib indentation convention?

Reply:
In that example, `(b : A)` should be indented. [Here's a link to the style guide.](https://leanprover-community.github.io/contribute/style.html)

Reply:
That document says nothing about line-wrapping arguments to definitions, nor could I see any examples of a wrapped definition within - would you mind updating it so that it does?

Reply:
Good point, arguably the `theorem nat_discriminate` example shows how to wrap lines with arguments, but I can make it explicit.

Reply:
[leanprover-community.github.io#143](https://github.com/leanprover-community/leanprover-community.github.io/pull/143/files)